Makefile

```diff
@@ -11,78 +15,109 @@ webbrowser.open("file://" + pathname2url(os.path.abspath(sys.argv[1])))
 endef
 export BROWSER_PYSCRIPT
 
+##################################
+# Display help for this makefile #
+##################################
 define PRINT_HELP_PYSCRIPT
 import re, sys
 
+print("BigchainDB Driver 0.5 developer toolbox")
+print("--------------------------------")
```

Comment:
`--------------------------------` → `---------------------------------------`

:stuck_out_tongue:

Reply:
and it's not even rst xD